src/main/java/uk/gov/hmcts/reform/professionalapi/Application.java

```diff
@@ -26,7 +25,6 @@
         "checkstyle:Indentation"}) // Spring needs a constructor, its not a utility class
 public class Application  {
     public static void main(final String[] args) {
-        ApplicationInsights.attach();
```

Comment:
Thanks for looking into this. Main question with the update if serenity reports are good enough, as rd-commons-lib was doing quite a lot to group, count and put stats into the report. Serenity itself improved reporting and might be good enough without customised analysis, that would simplify rd-commons-lib.

To the question you raised here, you can see that I also removed app insights from `web.config` file, that was using -javaagent parameter to pass some old version. Reason for those removals is that in Dockerfile there is this param set:
`ARG APP_INSIGHTS_AGENT_VERSION=3.5.2`
and that is used in the base java image to set -javaagent parameter for java, which is enough to enable app insights:
https://learn.microsoft.com/en-us/azure/azure-monitor/app/java-spring-boot#enabling-with-jvm-argument

Reply:
Thank you for pointing out the reporting impact.
https://build.hmcts.net/job/HMCTS_j_to_z/job/rd-professional-api/job/master/46/Functional_20Tests_20Report/
and
https://build.hmcts.net/job/HMCTS_j_to_z/job/rd-professional-api/job/PR-1708/2/Functional_20Tests_20Report/
are reporting 527 and 519 total tests respectively.
Is this caused by scenarios being counted differently or are some omitted?

Reply:
I see that both report 519 tests passing. In original reporting it shows 8 ignored tests that adds up to 527, but I can't find those ignored tests on master branch, so not sure where they are coming from. New reporting shows 0 ignored, suggesting that it would show ignored tests if there were any...

Reply:
Ok, I found what is happening. Whether test is ignored or not is controlled by `ToggleEnable` annotation and [professional-api has some logic](https://github.com/hmcts/rd-professional-api/blob/master/src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/util/CustomSerenityRunner.java#L37) to decide whether to disable/ignore test or not. So that is a change - new reporting doesn't see those ignored tests.